contrib/parseq-lambda-names/src/main/java/com/linkedin/parseq/lambda/SyntheticLambdaAnalyzer.java

```diff
@@ -64,6 +70,11 @@ String getInferredOperation() {
     }
 
     @Override
+    public void visitLineNumber(int line, Label start) {
+      _lineNumber = line;
```

Comment:
As a safety precaution, should we make this assignment just once to grab only the very first line number in case generated synthetic lambda method contains more than one line numbers?